flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/gateway/local/LocalExecutorITCase.java

```diff
@@ -211,6 +211,63 @@ public void testListDatabases() throws Exception {
 		executor.closeSession(sessionId);
 	}
 
+	@Test
+	public void testCreateDatabase() throws Exception {
+		final Executor executor = createDefaultExecutor(clusterClient);
+		final SessionContext session = new SessionContext("test-session", new Environment());
+		String sessionId = executor.openSession(session);
+		assertEquals("test-session", sessionId);
+
+		executor.executeUpdate(sessionId, "create database db1");
+
+		final List<String> actualDatabases = executor.listDatabases(sessionId);
+		final List<String> expectedDatabases = Arrays.asList("default_database", "db1");
+		assertEquals(expectedDatabases, actualDatabases);
+
+		executor.closeSession(sessionId);
+	}
+
+	@Test
+	public void testDropDatabase() throws Exception {
+		final Executor executor = createDefaultExecutor(clusterClient);
+		final SessionContext session = new SessionContext("test-session", new Environment());
+		String sessionId = executor.openSession(session);
+		assertEquals("test-session", sessionId);
+
+		executor.executeUpdate(sessionId, "create database db1");
+
+		List<String> actualDatabases = executor.listDatabases(sessionId);
+		List<String> expectedDatabases = Arrays.asList("default_database", "db1");
+		assertEquals(expectedDatabases, actualDatabases);
+
+		executor.executeUpdate(sessionId, "drop database if exists db1");
+
+		actualDatabases = executor.listDatabases(sessionId);
+		expectedDatabases = Arrays.asList("default_database");
+		assertEquals(expectedDatabases, actualDatabases);
+
+		executor.closeSession(sessionId);
+	}
+
+	@Test
+	public void testAlterDatabase() throws Exception {
+		final Executor executor = createDefaultExecutor(clusterClient);
+		final SessionContext session = new SessionContext("test-session", new Environment());
+		String sessionId = executor.openSession(session);
+		assertEquals("test-session", sessionId);
+
+		executor.executeUpdate(sessionId, "create database db1 comment 'db1_comment' with ('k1' = 'v1')");
+
+		executor.executeUpdate(sessionId, "alter database db1 set ('k1' = 'a', 'k2' = 'b')");
+
+		final List<String> actualDatabases = executor.listDatabases(sessionId);
+		final List<String> expectedDatabases = Arrays.asList("default_database", "db1");
```

Comment:
can we also assert properties?

Reply:
We can not obtain properties from the executor directly and I add a todo item once we support describe database we can complete this test.